docker/entrypoint-uwsgi-dev.sh

```diff
@@ -9,14 +9,19 @@ cd /app
 #   cp dojo/settings/settings.dist.py dojo/settings/settings.py
 # fi
 
+# Full list of uwsgi options: https://uwsgi-docs.readthedocs.io/en/latest/Options.html
+# --lazy-apps required for debugging --> https://uwsgi-docs.readthedocs.io/en/latest/articles/TheArtOfGracefulReloading.html?highlight=lazy-apps#preforking-vs-lazy-apps-vs-lazy
+
 exec uwsgi \
   "--${DD_UWSGI_MODE}" "${DD_UWSGI_ENDPOINT}" \
   --protocol uwsgi \
   --wsgi dojo.wsgi:application \
   --enable-threads \
-  --processes ${DD_UWSGI_NUM_OF_PROCESSES:-2} \
-  --threads ${DD_UWSGI_NUM_OF_THREADS:-2} \
```

Comment:
not visible here, but the PR seems to hardcode it to 1 for both params

Reply:
I had issues when using uwsgi, debugpy works better with it when there's only one process and one thread. 

I did hardcode `1` and remove the variable substitution for `DD_UWSGI_NUM_OF_PROCESSES` and `DD_UWSGI_NUM_OF_THREADS`, because these variables were always set in the Dockerfile and the helm chart.

Otherwise, the debugging the container would have the 2 processes/2 threads that are an issue with uwsgi.